docs/contributing/security.md

```diff
@@ -1,5 +1,9 @@
 # Reporting security issues
 
+```{warning}
+Ensure you are viewing our [latest security policy](https://docs.wagtail.org/en/latest/contributing/security.html).
```

Comment:
I couldn't find a nicer way to reference the canonical latest version of a page. If there's some sphinx shorthand I don't know about, please tell me!